libecl/tests/ecl_nnc_export.c

```diff
@@ -231,6 +232,19 @@ void test_export(const char * name, bool have_tran_data) {
     for (int i =0; i < ecl_nnc_export_get_size( grid ); i++)
       test_assert_true( ecl_nnc_equal( &nnc_data1[i] , &nnc_data2[i] ));
 
+    {
+      ecl_nnc_geometry_type * nnc_geo = ecl_nnc_geometry_alloc( grid );
```

Comment:
could you add one line right above the `for` loop with
```c
    test_assert_int_equal(actual, ecl_nnc_geometry_size(nnc_geo));
```
where `actual` is some positive integer that makes the test pass?